### ISSUE TITLE
BenchmarkDotNet is not working with LinqPad

### ISSUE THREAD

**Issue by tym32167** [dotnet/BenchmarkDotNet#593]
Hi there. 
I red [this post](http://aakinshin.net/blog/post/bdn-v0_10_7/) and decided to check how BenchmarkDotNet behaves in LinqPad. 
I downloaded latest LinqPad beta v5.26.03 and latest BenchmarkDotNet nuget package 0.10.10

Code for testing

```
void Main()
{
	var summary = BenchmarkRunner.Run<Foo>();
}

public class Foo
{
	[Benchmark]
	public void Test()
	{
		for (var i = 0; i < 100; i++)
		{
			i++;
		}
	}
}
```

Output
```
// ***** BenchmarkRunner: Start   *****
// Found benchmarks:
//   Foo.Test: DefaultJob

// Validating benchmarks:
Assembly LINQPadQuery which defines benchmarks references non-optimized LINQPad
Assembly LINQPadQuery, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null is located in temp. If you are running benchmarks from xUnit you need to disable shadow copy. It's not supported by design.
// * Artifacts cleanup *
```

Switching compiler optimizations did not help
Switching 'do not shadow assembly references' option  also did not help.

**Comment by adamsitnik** [comment]
Hello @tym32167 

This issue has been already reported and fixed (see #580)

Please download latest version (`0.10.10.350`) from our CI feed, it has the fix:

```xml
<packageSources>
  <add key="bdn-nightly" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
</packageSources>
```

**Comment by tym32167** [comment]
Hi, @adamsitnik . Thanks for info.
